articles/azure-maps/power-bi-visual-get-started.md

```diff
@@ -89,25 +86,26 @@ The following data buckets are available in the **Fields** pane of the Azure Map
 
 | Field     | Description  |
 |-----------|--------------|
+| Location  | Used to enter easily understandable geographical data such as country, state, and city.  |
 | Latitude  | The field used to specify the latitude value of the data points. Latitude values should be between -90 and 90 in decimal degrees format.  |
 | Longitude | The field used to specify the longitude value of the data points. Longitude values should be between -180 and 180 in decimal degrees format.  |
-| Legend    | The field used to categorize the data and assign a unique color for data points in each category. When this bucket is filled, a **Data colors** section will appear in the **Format** pane that allows adjustments to the colors. |
+| Legend    | The field used to categorize the data and assign a unique color for data points in each category. When this bucket is filled, a **Data colors** section appears in the **Format** pane that allows adjustments to the colors. |
 | Size      | The measure used for relative sizing of data points on the map.   |
 | Tooltips  | Other data fields to display in tooltips when shapes are hovered. |
 
 ## Map settings
 
 The **Map settings** section of the **Format** pane provide options for customizing how the map is displayed and reacts to updates.
 
-The **Map settings** section is divided into three subsections: [style](#style), [view](#view) and [controls](#controls).
+The **Map settings** section is divided into three subsections: [style], [view] and [controls].
 
 ### Style
 
 The following settings are available in the **Style** section:
 
 | Setting     | Description  |
 |-------------|--------------|
-| Style       | The style of the map. The dropdown list contains [greyscale light][gs-light], [greyscale dark][gs-dark], [night][night], [road shaded relief][RSR], [satellite][satellite] and [satellite road labels][satellite RL]. |
+| Style       | The style of the map. The dropdown list contains [greyscale light], [greyscale dark], [night], [road shaded relief], [satellite] and [satellite road labels]. |
```

Comment:
The dropdown list now contains different styles now: 
<img width="143" alt="image" src="https://user-images.githubusercontent.com/18752396/235638011-2d3db1c3-d90f-432f-be3c-c0cbf5a5954a.png">

@deniseatmicrosoft Some of the styles in the style pickers are not aligned with [our documentation](https://learn.microsoft.com/en-us/azure/azure-maps/supported-map-styles). Do you know who can help us confirm these?

Reply:
There are two in the list that do not appear in our docs anywhere: 
1. Hybrid 
2. Terra

Also, **grayscale (light)** appears twice.

<table>
<tr><td>Correct List (In order):</td>      <td>Current list:</td><td></td></tr>
<tr><td>blank</td>				<td>grayscale light</td><td></td></tr>
<tr><td>blank accessible</td>	<td>road</td><td></td></tr>
<tr><td>grayscale dark</td>		<td>satellite</td><td></td></tr>
<tr><td>&nbsp;</td>			    <td>Hybrid</td> <td><-- Needs removed (Bing maps only) </td></tr>
<tr><td>grayscale light</td>	<td>grayscale (dark)</td><td></td></tr>
<tr><td>high contrast dark</td>	<td>&nbsp;</td><td></td></tr>
<tr><td>&nbsp;</td>				<td>grayscale (light)</td><td><-- Needs removed (Duplicate)</td></tr>		    
<tr><td>night</td>	            <td>&nbsp;</td><td></td></tr>
<tr><td>&nbsp;</td>				<td>Terra</td><td><-- Needs removed (Not a valid option)</td></tr>
<tr><td>road</td>				<td>blank</td><td></td></tr>>
<tr><td>&nbsp;</td>				<td>blank (accessible)</td><td></td></tr>
<tr><td>road shaded relief</td>	<td>&nbsp;</td><td><-- Needs added (Missing from list)</td></tr>	
<tr><td>satellite</td>			<td>high contrast (dark)</td><td></td></tr>
<tr><td>satellite road labels</td>	<td>&nbsp;</td><td><-- Needs added (Missing from list)</td></tr>
<tr><td>&nbsp;</td>				<td>high contrast (light)</td><td></td></tr>
</table>

Reply:
![image](https://github.com/MicrosoftDocs/azure-docs/assets/33468793/41e5fe57-8881-4445-9d99-09af12ca0e8b)